lib/gtfs/route.rb

```diff
@@ -2,8 +2,8 @@ module GTFS
   class Route
     include GTFS::Model
 
-    has_required_attrs :id, :short_name, :long_name, :type
-    has_optional_attrs :agency_id, :desc, :url, :color, :text_color
+    has_required_attrs :id, :type
+    has_optional_attrs :agency_id, :desc, :url, :color, :text_color, :short_name, :long_name
```

Comment:
This should clear the merge conflict.

```suggestion
    has_optional_attrs :agency_id, :desc, :url, :color, :text_color, :sort_order, :short_name, :long_name
```